indico/modules/events/registration/forms.py

```diff
@@ -185,6 +185,17 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
+class RegistrationExceptionalModificationForm(IndicoForm):
+    modification_end_dt = IndicoDateTimeField(_('Modification deadline'), [Optional()], default_time=time(23, 59),
+                                              description=_('Deadline until which registration information can be '
+                                                            'modified (Will not allow modification if empty)'))
```

Comment:
Good point, I was going for flexibility for the operator to not have to close the form and click `close`, but just making it to be required makes more sense.

For now setting a deadline shorter than the event default won't make the deadline shorter, it only extends the deadline. I think that makes more sense since the original idea was to provide a way to extend the deadline, and I can't imagine someone would want to restrict certain participants.